.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -460,6 +463,16 @@ def write(x=''):
             }
         }
 
+        if args.setup_for_sample_courses:
+            worker_dict['primary']['capabilities'].extend([
+                'cpp',
+                'python',
+                'matlab',
+                'pdf-word-count',
```

Comment:
this does not look like it's used?

Reply:
agree...  only have the tags we use in the sample configs